assets/sass/modules/_btns.scss

```diff
@@ -88,7 +88,11 @@ $od-secondary-button-action-text-color: #124a94;
   line-height: 50px;
   margin-top: 15px;
   padding-left: 90px;
+  padding-right: 40px;
```

Comment:
we don't seem to have pre-defined variables or standards for margins/paddings hence why I hardcoded this value here.

Reply:
Not a designer, but shouldn't padding on the left and right be consistent so the content is centered properly?

Reply:
Yeah this is a good question. The answer is the buttons have the icon rendered on the left side and left padding takes this into account. It takes up about 50px according to what I saw hence why it's set to 40px. You can also refer to the video and see that the string is centered.

Reply:
Thanks for the great explanation! Hopefully we don't have any icon variants that push/pull this padding outside of the bounds we expect, but do agree this is a good approach for now.

Reply:
Yeah I don't think we'll have a case like that, at least by looking at this: https://github.com/okta/okta-signin-widget/blob/master/assets/sass/modules/_btns.scss#L198. According to that the background gradient color is set for a 50px X 50px square.